values.yaml

```diff
@@ -44,6 +44,10 @@ networkPolicies:
 # Used for mount paths
 airflowHome: "/usr/local/airflow"
 
+# Extra annotations to apply to all
+# Airflow pods
+airflowExtraAnnotations: {}
```

Comment:
@sjmiller609 how about `airflowPodAnnotations` to indicate that these actually go into the pod templates.